Podcast/ShareEpisodeViewController.swift

```diff
@@ -40,7 +40,7 @@ class ShareEpisodeViewController: FollowerFollowingViewController {
         let imageView = ImageView()
         imageView.setImageAsynchronouslyWithDefaultImage(url: episode.smallArtworkImageURL)
         if let image = imageView.image {
-            header = ActionSheetHeader(image: image, title: episode.title, description: episode.descriptionText)
+            header = ActionSheetHeader(image: image, title: episode.title, description: episode.attributedDescription.string)
```

Comment:
Wait is this the only place we show the description in the Action Sheet header? If so then we should probably just keep it consistent with the others and show title / dateTimeLabel

Reply:
done